src/branding/default/branding.desc

```diff
@@ -219,3 +219,18 @@ slideshow:               "show.qml"
 slideshowAPI: 2
 
 
+# These options are to customize online uploading of logs to pastebins:
+#  - style  :  Defines the kind of pastebin service to be used. ie it can
+#              provide the functionality of controlling privacy of 
+#              paste (in future).Currently only "fiche" servers
+#              are supported. Takes string as input
+#  - url    :  Defines the address of pastebin service to be used.
+#              Takes string as input
+#  - port   :  Defines the port number to be used to send logs. Takes 
+#              integer as input
+#  - enable :  Defines if the functionality is to be used or not. Takes 
+#              bool as input
```

Comment:
Tried creating YAML list here but was not able to call them properly in `branding.cpp`. So instead declared independent keys. Can be changed to following format in future : 
`logUpload:
        - style
        - url
        - port
        - enable`